packages/studio/src/index.ts

```diff
@@ -411,7 +411,7 @@ export type ExtensionPanelRegistration = {
 export type RegisterMessageConverterArgs<Src> = {
   fromSchemaName: string;
   toSchemaName: string;
-  converter: (msg: Src) => unknown;
+  converter: ((msg: Src) => unknown) | ((msg: Src, event: Immutable<MessageEvent<Src>>) => unknown);
```

Comment:
```suggestion
  converter: (msg: Src, event: Immutable<MessageEvent<Src>>) => unknown;
```

I think it should work this way. The additional argument shouldn't cause TS errors for people who pass in functions that only take 1 argument.

Reply:
Yes you're right that also seems to work.